File: BackEndAPI/Data/Entities/ChiTietDonHang.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackEndAPI.Entities
{
    public partial class ChiTietDonHang
    {
        public int Id { get; set; }
        public int MaDonHang { get; set; }
        public int MaSp { get; set; }
        public int? SoLuong { get; set; }
        public int? DonGia { get; set; }

        public virtual DonHang DonHang { get; set; }
        public virtual SanPham SanPham { get; set; }
    }
}

File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/ChiTietDonHang.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "CHI_TIET_DON_HANG")
public class ChiTietDonHang {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private int ma_don_hang;
    private int ma_sp;
    private int so_luong;
    private int don_gia;
}
